configurations/luminate_pipeline.py

```diff
@@ -0,0 +1,257 @@
+from core_data_modules.cleaners import swahili
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="Luminate",
```

Comment:
Is it worth calling this Luminate-S01 or is this likely to be the only series?

Reply:
Their might be another series, renamed to Luminate-S01 for now, thnaks.